modules/conversion-center/classes/render/core-render.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+namespace Elementor\Modules\ConversionCenter\Classes\Render;
+
+use Elementor\Modules\ConversionCenter\Widgets\Link_In_Bio;
+use Elementor\Utils;
+
+/**
+ * Class Core_Render.
+ *
+ * This class handles the rendering of the Link In Bio widget for the core version.
+ *
+ * @since 3.23.0
+ */
+class Core_Render extends Render_Base {
+
+	public function render( Link_In_Bio $widget ): void {
+		?>
+		<div class="e-link-in-bio">
+			<div class="e-link-in-bio__content">
+
+				<?php $this->render_identity_image( $widget ); ?>
+
+				<?php $this->render_bio( $widget ); ?>
+
+				<?php $this->render_icons( $widget ); ?>
+
+				<?php $this->render_ctas( $widget ); ?>
```

Comment:
@nicoladj77 Can you add a CTOR to this class that accepts the `$widget_instance` and `$settings` as arguments? This way you can access the widget methods from the class variable like `$this->widget_instance-> get_render_attribute_string()` and not keep passing it around to every function. And same with the settings, it will save many function calls to `get_settings_for_display()`

Reply:
No need for so many opening/closing PHP tags.
```
<?php
$this->render_identity_image( $widget );

$this->render_bio( $widget );

$this->render_icons( $widget );

 $this->render_ctas( $widget );
?>
```

Reply:
done